lib/instagram_basic_display/auth.rb

```diff
@@ -97,5 +97,13 @@ def refresh_long_lived_token(token:)
     private
 
     attr_reader :configuration
+
+    def auth_url
+      if configuration.version == 1
+        'https://api.instagram.com/oauth/access_token'
+      else
+        'https://graph.instagram.com/oauth/access_token'
+      end
+    end
```

Comment:
Feels like this could be a part of Configuration object, it would allow us to encapsulate the version logic there. What do you think?